options/locale/locale_en-US.ini

```diff
@@ -1274,6 +1274,7 @@ commit_graph.color = Color
 commit.contained_in = This commit is contained in:
 commit.contained_in_default_branch = This commit is part of the default branch
 commit.load_referencing_branches_and_tags = Load branches and tags referencing this commit
+commit.load_tags_failed = Load tags failed because of internal error
```

Comment:
Would there be a lot of duplicate (and meaningless) translations?  "Load users failed because of internal error"  "Load branches failed because of internal error" "Load workflows failed because of internal error" "Load releases failed because of internal error" "Load projects failed because of internal error" "Load labels failed because of internal error" "Load milestones failed because of internal error"

Reply:
I will send another PR to fix all of them.

Reply:
#31878